### DISCUSSION TITLE
I want to know whether promtail has modified the interface of yaml configuration file or added the interface of job target to it

### DISCUSSION THREAD

**Discussion by lishensheng** [prometheus-operator/prometheus-operator#4360] [Q&A]
I want to know whether promtail has modified the interface of yaml configuration file or added the interface of job target to it

**Comment by paulfantom** [comment]
Sorry, but I don't see how this question relates to prometheus-operator considering that `promtail` is a completely different project. Could you clarify your question?

**Comment by lishensheng** [comment]
I want to know if "promtail" has an interface that can directly manipulate the "promtail.yaml" file.Is it possible to restart the "promtail" service directly after modifying the "promtail-config.yaml" file


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "prometheus-operator/prometheus-operator"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月2日(星期二) 下午4:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [prometheus-operator/prometheus-operator] I want to know whether promtail has modified the interface of yaml configuration file or added the interface of job target to it (Discussion #4360)





 
Sorry, but I don't see how this question relates to prometheus-operator considering that promtail is a completely different project. Could you clarify your question?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by paulfantom** [reply]
That's a question for promtail developers at https://github.com/grafana/loki
